analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -596,6 +613,40 @@ def construct_config_handler(cls, args):
             cmdline_checkers,
             'enable_all' in args and args.enable_all)
 
+        # If --stats flag is provided then enable statistics-based checkers
+        # unless explicitly disabled by the user.
+        if 'stats_enabled' in args:
+            # In some Clang versions, statisticsbased checkers are in the alpha
+            # package (alpha.statisticsbased), and maybe not in others. Lets
+            # figure out the actual checker name first.
+            special_return_value = next((
+                checker for checker, _ in checkers
+                if SpecialReturnValueCollector.checker_analyze in checker),
+                None)
+            unchecked_return_value = next((
+                checker for checker, _ in checkers
+                if ReturnValueCollector.checker_analyze in checker),
+                None)
+
+            if special_return_value and not _is_user_disabled_checker(
+                    special_return_value,
+                    cmdline_checkers):
+                handler.set_checker_enabled(special_return_value)
+
+            if unchecked_return_value and not _is_user_disabled_checker(
+                    unchecked_return_value,
+                    cmdline_checkers):
+                handler.set_checker_enabled(unchecked_return_value)
+
+        # Statistics collector checkers must be explicitly disabled as they
+        # trash the output. Thise checkers are executed in a preceding analysis
```

Comment:
```suggestion
        # trash the output. These checkers are executed in a preceding analysis
```

Reply:
Also, almost there, how about you end the sentence with "so they are not needed here."